### DISCUSSION TITLE
No combined charts on the dashboard

### DISCUSSION THREAD

**Discussion by svenvg93** [alexjustesen/speedtest-tracker#1349] [Q&A] [answered]
Hi all, 

Love this project, thanks for all the amazing work. I do have one question, in all the docs I see the download / upload charts combined. Somehow in mine install (latest LSIO version) the charts are separeate. Did I somewhere miss a settings to combine them, or is it changed and they cant be combined anymore. 

Its like this:
![image](https://github.com/alexjustesen/speedtest-tracker/assets/4511676/ba60e79b-864f-4fa2-be44-70f4739f2973)
 
instead of this: 
![image](https://github.com/alexjustesen/speedtest-tracker/assets/4511676/34398fec-f900-4e7a-be16-7379fa6fa906)

**Comment by alexjustesen** [comment, marked as answer]
Nah, docs just need to be updated. This was separated on purpose a few releases ago for those that have large gaps in their upload vs. download stats.

**Reply by svenvg93** [reply]
Shame I liked it more ;). Thanks for the awesome tool!
